Quartz/Engine/Source/Core/Graphics/API/GL/GLShaderPipeline.cpp

```diff
@@ -151,15 +163,36 @@ namespace
 			if (directive == "include") 
 			{
 				std::string filename = getNextToken();
+				if (filename.size() > 2)
+				{
 
-				filename.pop_back(); // remove last "
-				filename.erase(filename.begin());
-
-				std::string path = getDirname(shaderfilepath);
-				std::string filepathtolookfor = path + filename;
-
+					filename = filename.substr(1, filename.size() - 2);// remove last " and leading "
+				}
+				else
+				{
+					LFATAL("Shader source code error at %d : %d Expected filename after include!", linenum, colnum);
```

Comment:
It's been mentioned on discord, but it's probably worth removing the format string stuff here for now (until we have a logger that supports them).

The current logger will just concatenate any arguments passed to the macro, so:
`LFATAL("Hello", " ", "World"); // Log's "Hello World"`
`LFATAL("Some Number: ", 23, ", and something else: ", 'a') // Log's "Some Number: 23 and something else: a"`.

It's fairly shitty compared to format strings but 🤷‍♂